core/aggregation.go

```diff
@@ -361,16 +367,18 @@ func GetStakeThreshold(state *OperatorState, quorum QuorumID, quorumThreshold ui
 	return stakeThreshold
 }
 
-func GetSignedPercentage(state *OperatorState, quorum QuorumID, stakeAmount *big.Int) uint8 {
+func GetSignedPercentage(state *OperatorState, quorum QuorumID, stakeAmount *big.Int) (uint8, error) {
 	totalStake := state.Totals[quorum].Stake
 	if totalStake.Cmp(big.NewInt(0)) == 0 {
-		return 0
+		return 0, nil
 	}
 
 	stakeAmount = stakeAmount.Mul(stakeAmount, new(big.Int).SetUint64(percentMultiplier))
 	quorumThresholdBig := stakeAmount.Div(stakeAmount, totalStake)
 
 	quorumThreshold := uint8(quorumThresholdBig.Uint64())
-
-	return quorumThreshold
+	if quorumThreshold > 100 {
```

Comment:
why this needs to check? It's just computed locally and it has fully context about the calculation, which the caller may not know even if the error is returned